py/torch_tensorrt/dynamo/backend/backends.py

```diff
@@ -77,13 +77,12 @@ def _pretraced_backend(
         )
         return trt_compiled
     except:
-        logger.error(
-            "FX2TRT conversion failed on the subgraph. See trace above. "
-            + "Returning GraphModule forward instead.",
-            exc_info=True,
-        )
-
         if not settings.pass_through_build_failures:
+            logger.error(
```

Comment:
What is typically the policy for compilation failures in torch.compile? should this be an error or a warning or info since this is part of expected behavior for torch.compile

Reply:
I don't think there is a formal policy for compilation failures in `torch.compile`, it is more the choice of the backend. It seems that many other backends (such as `onnxrt`, `tvm`) generally do not have `try/except` clauses in their highest-level backends to catch internal errors, from what I could see. In this case, I think `logger.error` would be the correct level of error to report.

Reply:
Switch to a `warning` + still display error from TRT. Fallback to PyTorch.